docs/installing.md

```diff
@@ -1,104 +1,58 @@
 # Running a local Haveno test network
 
-These are the steps needed to set up a local Haveno test network.
+ These are the steps needed to set up a local Haveno test network.  You'll have the possibility to either connect to our shared Monero stagenet node or to create your own local environment.
```

Comment:
Unecessary whitespace at start.

2 space indent between sentences.

They'll have the opportunity to connect to our shared Monero stagenet or create their own private stagenet. The environment is local regardless.